src/app/modules/qsa/components/modals/calculation/calculation.modal.html

```diff
@@ -0,0 +1,33 @@
+<div class="modal-header">
+    <h5 class="modal-title">Calculated using the following formula</h5>
```

Comment:
Please use the i18n source to handle internationalisation. This applies to other texts as well.